.devops/templates/variables.yml

```diff
@@ -1,20 +1,13 @@
-parameters:
-  - name: pathPrefix
-    type: string
-    default: SourceBranch
-    values:
-      - SourceBranch # matches branch/PR ref
-      - SourceVersion # matches commit SHA
-
 variables:
-  ${{ if eq( parameters.pathPrefix, 'SourceBranch' ) }}:
-    deployBasePath: "pr-deploy-site/${{ variables['Build.SourceBranch'] }}"
-  ${{ if eq( parameters.pathPrefix, 'SourceVersion' ) }}:
-    deployBasePath: "pr-deploy-site/${{ variables['Build.SourceVersion'] }}"
```

Comment:
This was intentional to avoid collisions between N* Screener task and build artifacts. Can we still do something similar?

Reply:
Could you explain more about what collision you're concerned about? If we do need some additional special-purpose path, I'd prefer to put it in a subfolder under the PR's main deployed folder (instead of using the hash) so we don't clutter the `pr-deploy-site` folder when looking at the storage account contents. (Though granted I suspect that mainly just affects me, since most of the team probably doesn't spend much time digging around the storage account.)

Reply:
I moved the screener deploy to `pr-deploy-site/<Build.SourceBranch>/react-northstar-screener`

Reply:
And now updated the 404 page to handle the new path as well

Reply:
Now it's simplified, under `pull/####/react-northstar-screener` for PRs or `heads/branchname/react-northstar-screener` for CI